code-challenges.test.js

```diff
@@ -40,11 +80,56 @@ const letterE = "e"
 
 // b) Create the function that makes the test pass.
 
+// const wordFinder = (array, string) => {
+//     let find = array.filter(value => {
+//         value.includes(find)
+//         return string.split(" ")
+//     })
+// }
```

Comment:
This seems to be heading in the right direction.  A few small changes and you should be passing your test.  Keep in mind that includes will check for a specific value (ex: string) if it is in the value.  Also, make sure to pass both the array and string into the function when invoking in your test.